customasset.cpp

```diff
@@ -81,13 +81,13 @@ CustomAsset::~CustomAsset()
  * @param triggerReason		Why the notification is being sent
  * @param message		The message to send
  */
-void CustomAsset::notify(const string& notificationName, const string& triggerReason, const string& message)
+bool CustomAsset::notify(const string& notificationName, const string& triggerReason, const string& message)
 {
 	vector<Datapoint *>	datapoints;
 
 	if (!m_ingest)
 	{
-		return;
+		return false;
```

Comment:
Can we log some message here why retuning false ?

Reply:
fixed